bot.py

```diff
@@ -147,28 +175,16 @@ async def remove(ctx):
             }
         )
     )
-    remove_from = await bot.wait_for("message", timeout=60)
-    if remove_from:
-        tracker.remove_inv(author, remove_from.content)
-
-
-@bot.command()
-async def inv(ctx):
-    author = ctx.message.author.id
-    name = ctx.message.author.name
-    one_line = "\n".join(tracker.get_inv(author))
-    await ctx.message.channel.send(
-        embed = Embed().from_dict(
-            {
-                "fields": [
-                    {
-                        "name": f"__*{name}'s Inventory:*__",
-                        "value": one_line,
-                    }
-                ]    
-            }
-        )
-    )
+    while True:
+        try:
+            remove_message = await bot.wait_for("message", timeout=60)
+            if ctx.message.channel.id == remove_message.channel.id and author == remove_message.author.id:
+                tracker.remove_inv(author, remove_message.content)
+                await remove_message.add_reaction("❎")
+                break
+        except:
+            await ctx.message.channel.send(f"Inventory remove request timed out. Please use {bot_prefix}inv remove again.")
+            break
```

Comment:
Same as `!inv add`, we can rely on `asyncio.TimeoutError` to simplify the code a bit.

```suggestion
    while True:
        try:
            remove_message = await bot.wait_for("message", timeout=60)
        except TimeoutError:
            await ctx.message.channel.send(
                f"Inventory remove request timed out. Please use `{bot_prefix}inv` add again."
            )
            break

        remove_message = await bot.wait_for("message", timeout=60)
        if (
            ctx.message.channel.id == remove_message.channel.id
            and author == remove_message.author.id
        ):
            tracker.remove_inv(author, remove_message.content)
            await remove_message.add_reaction("❎")
            break
```

Reply:
Also same comments here with `author` vs `author_id`.

Reply:
Suggested changes made. Thanks!